src/emulator-manager.ts

```diff
@@ -27,6 +27,9 @@ export async function launchEmulator(
     `sh -c \\"echo no | avdmanager create avd --force -n "${avdName}" --abi '${target}/${arch}' --package 'system-images;android-${apiLevel};${target};${arch}' ${profileOption} ${sdcardPathOrSizeOption}"`
   );
 
+  // set number of cores to 2 (instead of 1 because there are 3 cores on the Github-hoster macOS VM)
+  await exec.exec(`sh -c \\"printf 'hw.cpu.ncore=2\n' >> ~/.android/avd/"${avdName}".avd"/config.ini`);
```

Comment:
Could this be configurable, too, with some kind of default (i.e., do nothing)? I feel this is the only change that doesn't generalize as is to submit upstream.

Reply:
Had a discussion with the maintainer about this change once I figured this was the difference between our Azure pipeline and the Emulator on GitHub. We can definitely generalize it though.

https://github.com/ReactiveCircus/android-emulator-runner/issues/126

Reply:
I was going to suggest exactly this. 👏

Reply:
Done! There's a `cores` option now :)

Reply:
Great investigation @AfzalivE. :raised_hands: